app/scripts/modules/core/src/presentation/illustrations/illustrationsByName.ts

```diff
@@ -7,6 +9,8 @@ import { ReactComponent as unpinArtifactVersion } from './vectors/unpinArtifactV
 
 export const illustrationsByName = {
   announcement,
+  appSynced,
+  appUnsynced,
```

Comment:
is this illustration referenced via an `<Illustration/>` component somewhere already? I don't see it in the diff.

Reply:
@erikmunson Not yet, I have an upcoming PR that will integrate it with some of the layout changes: https://github.com/spinnaker/deck/pull/8556/files#diff-e446ec3f6fb430ca891053eadf01aec6R10

Reply:
👍 👍